ui/app/pages/token/index.scss

```diff
@@ -0,0 +1,148 @@
+.token {
```

Comment:
Was this meant to go on the  `add-token` page? It's just a lone `.scss` file at the moment.

Reply:
This is a new directory for an upcoming feature. Currently this css is used in `/home/danjm/kyokan/metaswaps/ui/app/components/ui/searchable-item-list/searchable-item-list.stories.js`

Reply:
It seems unnecessary to include additional features and styles in what is already quite a large refactoring PR.